tests/vmi_multus_test.go

```diff
@@ -862,16 +862,24 @@ var _ = Describe("[Serial]SRIOV", func() {
 			cidrA := "192.168.1.1/24"
 			cidrB := "192.168.1.2/24"
 			vmi1, vmi2 := createSriovVMs(cidrA, cidrB)
-			Expect(libnet.PingFromVMConsole(vmi1, cidrToIP(cidrB))).To(Succeed())
-			Expect(libnet.PingFromVMConsole(vmi2, cidrToIP(cidrA))).To(Succeed())
+			Eventually(func() error {
+				return libnet.PingFromVMConsole(vmi1, cidrToIP(cidrB))
+			}, 15*time.Second, time.Second).Should(Succeed())
+			Eventually(func() error {
```

Comment:
Do we want to add a `Consistently` after the `Eventually` 
to see the ping is stable ? (it tries just 1 packet atm, or at least to make it ping with more packets?)

Reply:
I remember we did it on the network policy test there it did make sense since we didn't know if network policy was really applied or not, this case is different if one ICMP package is able to go from VM to VM then we good.